tests/resources_t/test_drone.py

```diff
@@ -53,6 +54,11 @@ def test_demand(self):
         self.drone.demand = 0
         self.assertEqual(self.drone.demand, 0)
 
+    def test_heartbeat_interval(self):
```

Comment:
Generic comment about the unit tests. The tests currently only target checking if the property has been set. It would be great to add some edge cases such as negative values and 0 here. Further if it is not too complicated I would really appreciate having a test that actually tests if the heartbeat is used at all.

Reply:
Agreed to a add the edge cases to https://github.com/MatterMiners/tardis/blob/e4e35068c1c083a40ef0f0a878bd9f5c93e6545f/tests/resources_t/test_drone.py#L86. Also including a test that checks that `async.sleep` is actually called should be easy to add here.

Reply:
Added a check that `async.sleep` is call with the heartbeat as argument. Edge cases are tested in the site adapter interface unittests.

Reply:
That's really amazing how you extended the unit tests! 👍